app/views/admin/submissions/_note.html.erb

```diff
@@ -0,0 +1,10 @@
+  <div class='list-group-item--note'>
```

Comment:
I'm not sure i did my bem classes or used [artsy/watt](https://github.com/artsy/watt) right.